tests/test_cli_claude_teams_env.py

```
@@ -203,13 +203,13 @@ def run_claude_teams(
         term_value = read_text(term_log)
-        if term_value != "screen-256color":
-            print(f"FAIL: expected TERM=screen-256color, got {term_value!r}")
+        if term_value != "xterm-256color":
+            print(f"FAIL: expected TERM=xterm-256color, got {term_value!r}")
             raise SystemExit(1)
 
         term_program_value = read_text(term_program_log)
-        if term_program_value != "__UNSET__":
-            print(f"FAIL: expected TERM_PROGRAM to be unset, got {term_program_value!r}")
+        if term_program_value == "__UNSET__":
+            print(f"FAIL: expected TERM_PROGRAM to be preserved, got unset")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)